Documentation/project-docs/target-dotnetcore-with-msbuild.md

```diff
@@ -0,0 +1,119 @@
+[coretemplate](https://github.com/mellinoe/coretemplate).  This is a basic project which brings in MSBuild targets for .NET Core.  
+
+It is also possible to start by creating a project in Visual Studio and modify it to target .NET Core.  The instructions below show the minimal steps to get this working.
+In contrast to coretemplate, a project created this way:
+
+- Won't include configurations for targeting Linux and Mac OS
+- Will use the CoreRun instead of the CoreConsole host.  This means that you will need to run the app via `CoreRun MyApp.exe` instead of just `MyApp.exe`.
+To use the CoreConsole host, you need to reference the `Microsoft.NETCore.ConsoleHost` package instead of `Microsoft.NETCore.TestHost`, and after building rename `MyApp.exe`
+to `MyApp.dll` and rename `CoreConsole.exe` to `MyApp.exe`.  Coretemplate does this in a [common targets file](https://github.com/mellinoe/corebuild/blob/master/coreconsole.targets).
+- Visual Studio may ask you to enable developer mode for Windows 10 when you open the project
+
+Creating a .NET Core console application from Visual Studio
+===========================================================
+
+- Start by following the steps in the previous section to create a library targeting .NET Core
+- Open the project's XML for editing (in Visual Studio, right click on the project -> Unload Project, right click again -> Edit MyProj.csproj)
+    - Remove the `ProjectTypeGuids` property
+    - Add the following property: `<BaseNuGetRuntimeIdentifier>win10</BaseNuGetRuntimeIdentifier>`
+    - Add the following at the end of the file (after the import of `Microsoft.Portable.CSharp.Targets`:
+
+        ```xml
+        <PropertyGroup>
+            <AutoUnifyAssemblyReferences>true</AutoUnifyAssemblyReferences>
+            <StartAction>Program</StartAction>
+            <StartProgram>$(TargetDir)CoreRun.exe</StartProgram>
+            <StartArguments>$(TargetPath)</StartArguments>
+            <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
+        </PropertyGroup>
+        ```
+
+    - Close the .csproj file, and reload the project in Visual Studio
+- In the project properties:
+    - Change the Output type to "Console Application" in the Application tab
+    - In the "Build" tab, select "All Configurations" and change the "Platform Target" to "x64"
+- Add runtimes and dependencies on the runtime and host to `project.json`
+    - Add a `runtimes` section with the following runtimes: `win10-x64`, `ubuntu.14.04-x64`, and `osx.10.10-x64`
+    - Add a dependency on `Microsoft.NETCore.Runtime` version `1.0.2-rc2-24027` and `Microsoft.NETCore.TestHost` version `1.0.0-rc2-24027`
+    - When you're done, the `project.json` should look like this:
+
+        ```json
+        {
+            "dependencies": {
+                "NETStandard.Library": "1.5.0-rc2-24027",
+                "Microsoft.NETCore.TestHost": "1.0.0-rc2-24027",
+                "Microsoft.NETCore.Runtime": "1.0.2-rc2-24027"
```

Comment:
I do think we should use Microsoft.NETCore.App but if you stick with the runtime specifically you should switch to "Microsoft.NETCore.Runtime.CoreCLR" as we are removing "Microsoft.NETCore.Runtime" package for RTM.